core/domain/question_domain.py

```diff
@@ -1408,6 +1408,24 @@ def _convert_state_v51_dict_to_v52_dict(
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v52_dict_to_v53_dict(
```

Comment:
Are we required to change the `question` part? and if so then I think we also require to edit the links here, right?
I think @vojtechjelinek can also help us here.

Reply:
I think we should also remove the invalid links from the questions, so the code will be quite similar here.

Reply:
Done